This is somehow incompatible with newlib's <sys/cdefs.h>, which kicks
in when any standard header is #include'd first (e.g. in bindings/, but
not src/internal.h, where this is first).

--- origsrc/getdata-0.8.5/src/getdata.h.in	2013-04-05 19:13:57.000000000 -0500
+++ src/getdata-0.8.5/src/getdata.h.in	2013-10-13 23:28:43.128556400 -0500
@@ -102,10 +102,10 @@ extern "C" {
 /* ICC complains that it ignores the __nonnull__ attribute */
 #ifdef __INTEL_COMPILER
 # define gd_nonnull(...) /**/
-#elif defined __nonnull
-# define gd_nonnull __nonnull
 #elif gd_gnu_enough
 # define gd_nonnull(...) __attribute__ ((__nonnull__ __VA_ARGS__))
+#elif defined __nonnull
+# define gd_nonnull __nonnull
 #else
 # define gd_nonnull(...) /**/
 #endif
